qiskit_nature/second_q/operators/commutators.py

```diff
@@ -0,0 +1,107 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Commutator Utilities (:mod:`qiskit_nature.second_q.operators.commutators`)
+===================================================================================
+
+.. currentmodule:: qiskit_nature.second_q.operators.commutator
+
+Utility functions to generate compute commutators for
+:class:`qiskit_nature.second_q.operators.sparse_label_op`.
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   commutator
+   anti_commutator
+   double_commutator
+
+"""
+
+from .sparse_label_op import SparseLabelOp
+
+
+def commutator(op_a: SparseLabelOp, op_b: SparseLabelOp) -> SparseLabelOp:
+    r"""
+    Compute commutator of `op_a` and `op_b`.
+    .. math::
+        AB - BA.
+    Args:
+        op_a: Operator A
+        op_b: Operator B
+    Returns:
+        SparseLabelOp: the commutator
+    """
```

Comment:
```suggestion
    r"""Compute commutator of `op_a` and `op_b`.

    .. math::

        AB - BA.

    Args:
        op_a: Operator A
        op_b: Operator B

    Returns:
        SparseLabelOp: the commutator
    """
```

Reply:
Similar reformatting is required in the other docstrings

Reply:
And you will need to add exceptions to the `.pylintdict` file to ignore the spelling of `AB`, etc.
Please add those exceptions in all lowercase and at the correct alphabetical position in that file